hawc/apps/animal/constants.py

```diff
@@ -0,0 +1,144 @@
+from django.db import models
+
+
+class ExperimentType(models.TextChoices):
+    AC = "Ac", "Acute (<24 hr)"
+    ST = "St", "Short-term (1-30 days)"
+    SB = "Sb", "Subchronic (30-90 days)"
+    CH = "Ch", "Chronic (>90 days)"
+    CA = "Ca", "Cancer"
+    ME = "Me", "Mechanistic"
+    RP = "Rp", "Reproductive"
+    R1 = "1r", "1-generation reproductive"
+    R2 = "2r", "2-generation reproductive"
+    DV = "Dv", "Developmental"
+    OT = "Ot", "Other"
+    NR = "NR", "Not-reported"
+
+
+class PurityQualifier(models.TextChoices):
+    GT = ">", ">"
+    GTE = "≥", "≥"
+    EQUALS = "=", "="
+    NA = "", ""
+
+
+class Sex(models.TextChoices):
+    M = "M", "Male"
+    F = "F", "Female"
+    C = "C", "Combined"
+    R = "R", "Not reported"
+
+
+class Generation(models.TextChoices):
+    NA = "", "N/A (not generational-study)"
+    P0 = "P0", "Parent-generation (P0)"
+    F1 = "F1", "First-generation (F1)"
+    F2 = "F2", "Second-generation (F2)"
+    F3 = "F3", "Third-generation (F3)"
+    F4 = "F4", "Fourth-generation (F4)"
+    OT = "Ot", "Other"
+
+
+# these choices for lifesage expose/assessed were added ~Sept 2018. B/c there
```

Comment:
put this comment in a [triple-quoted string w/ numpy format](https://github.com/numpy/numpy/blob/v1.21.0/numpy/lib/npyio.py#L1202-L1447)